roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,22 +15,37 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
         },
+        "online": {
```

Comment:
I believe the comment I linked refers to RPM packages with a prefix of "openshift-"... I don't know / didn't look for where we define what images are used when `deployment_type` is `online`.

Once we know that we can also update https://github.com/openshift/openshift-ansible/pull/3799.

Why did you start with setting this to the same as origin? I thought you may had found it somewhere :)